recipe/meta.yaml

```diff
@@ -73,6 +74,50 @@ outputs:
         - sqlalchemy <3,>=1.4
         - sqlalchemy-views <1,>=0.3.1
         - {{ pin_subpackage(name + '-core') }}
+        # - typing_extensions <5,>=4.3.0
+        # - trino-python-client <1,>=0.321
+        # - toolz <1,>=0.11
+        # - sqlglot <20,>=18.12.0
+        # - snowflake-sqlalchemy <2,>=1.4.1
+        # - snowflake-connector-python !=3.3.0b1,<4,>=3.0.2
+        # - shapely <3,>=2
+        # - rich <14,>=12.4.4
+        # - requests <3,>=2
+        # - regex >=2021.7.6
+        # - pytz >=2022.7
+        # - python-graphviz <1,>=0.16
+        # - python-dateutil <3,>=2.8.2
+        # - pyspark <3.4,>=3
+        # - pymysql <2,>=1
+        # - pymssql <3,>=2.2.5
+        # - pydruid <1,>=0.6.5
+        # - pydata-google-auth <2,>=1.4.0
+        # - pyarrow-hotfix <1,>=0.4
+        # - pyarrow <15,>=2
+        # - psycopg2 <3,>=2.8.4
+        # - polars <1,>=0.19.3
+        # - pins <1,>=0.8.3
+        # - parsy <3,>=2
+        # - pandas <3,>=1.2.5
+        # - packaging <24,>=21.3
+        # - oracledb <2,>=1.3.1
+        # - numpy <2,>=1
+        # - multipledispatch <2,>=0.6
+        # - impyla <1,>=0.17
+        # - google-cloud-bigquery-storage <3,>=2
+        # - google-cloud-bigquery <4,>=3
+        # - geopandas <1,>=0.6
+        # - geoalchemy2 <1,>=0.6.3
+        # - fsspec >=2022.1.0
+        # - filelock <4,>=3.7.0
+        # - deltalake <1,>=0.9.0
+        # - db-dtypes <2,>=0.3
+        # - datafusion <33,>=0.6
+        # - dask-core >=2022.9.1
+        # - clickhouse-connect <1,>=0.5.23
+        # - black <24,>=22.1.0
+        # - bidict <1,>=0.22.1
+        # - atpublic <5,>=2.3
```

Comment:
Is this big comment list generated by the regro tick bot?

Reply:
yeah, i think it is, and I don't know why

Reply:
Do we want to rip this out? Or leave it for the moment, merge in this release, and then ask the conda-forge team what's up?

Reply:
I think this is fine for now